etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/AddContactSectionToPatientResource.java

```diff
@@ -12,8 +12,8 @@
 import org.hl7.fhir.r4.model.Coding;
 import org.hl7.fhir.r4.model.Patient;
 
-/** Custom transformation to add a contact section to a patient resource. */
-public class addContactSectionToPatientResource implements CustomFhirTransformation {
+/** Adds mother's contact section to patient. */
+public class AddContactSectionToPatientResource implements CustomFhirTransformation {
```

Comment:
I know the try catch was mentioned before. Thoughts on adding logging?

Reply:
And to clarify, logging should only be used if the exception is being _completely_ handled.  E.g. if you re-throw a new exception from the caught exception, don't log.  Something else will eventually completely handle that exception and log at that point.

Reply:
Exception handling for transformations was refactored in PR 1104 and we no longer need try/catch here. In terms of adding logging, I think it could be useful to have debug level logging here, but info level logging may create too much noise. Unless there's something specific we need to log? Info logging for each transformation is already happening in the engine